CONTRIBUTING.org

```diff
@@ -83,14 +85,14 @@ Spacemacs is based on conventions, mainly for naming functions, keybindings
 definition and writing documentation. Please read the [[https://github.com/syl20bnr/spacemacs/blob/develop/doc/CONVENTIONS.org][CONVENTIONS.org]] file
 before your first contribution to get to know them.
 
-*** Pull-Request
+*** Pull request
 Submit your contribution against the =develop= branch. You should not use
 your =master= branch to modify Spacemacs, this branch is considered to be
 read-only.
 
-You may want to [[https://github.com/syl20bnr/spacemacs/wiki/Beginner%27s-Guide-to-Contributing-a-Pull-Request-to-Spacemacs][read our beginner’s guide for Pull Requests]].
+You may want to [[https://github.com/syl20bnr/spacemacs/wiki/Beginner's-Guide-to-Contributing-a-Pull-Request-to-Spacemacs][read our beginner’s guide for pull requests]].
```

Comment:
No.

Reply:
Thanks, didn't notice that org-mode changed the link when I edited the description! By the way, is there a specific reason for capitalising Pull Requests? I grepped the spacemacs project and different documents seem to have both uncapitalised and capitalised instances. Github does not capitalise, so I went with that.